docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -205,9 +207,17 @@ For more information, see xref:security-oauth2.adoc[Using OAuth2].
 
 [[oidc-jwt-oauth2-comparison]]
 
-== Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms
+== Choose the authentication mechanism
+
+Quarkus provides the following authentication mechanisms to secure your Quarkus applications:
```

Comment:
This IMO makes it look like it is an exhaustive list (and that Quarkus doesn't support anything beyond these 3), which isn't true. It's just that this section attempts to explain the differences between these 3 mechanisms in particular, but there are more (for example, just a hardcoded list of users and passwords).

Reply:
Hm, ok. Will put it back. Thank you